.pipelines/v2/templates/job-build-project.yml

```diff
@@ -376,6 +376,12 @@ jobs:
         msbuildArchitecture: x64
         maximumCpuCount: true
 
+  ### HACK: On ARM64 builds, building an app with Windows App SDK copies the x64 WebView2 dll instead of the ARM64 one. This command makes sure the right dll is used.
+  - script: |
+      COPY /Y "packages\Microsoft.Web.WebView2.1.0.2739.15\runtimes\win-ARM64\native_uap\Microsoft.Web.WebView2.Core.dll" "ARM64\Release\WinUI3Apps\Microsoft.Web.WebView2.Core.dll" || exit /b 1
```

Comment:
Since this is just a file copy, can we use the file copy task instead?

Reply:
I'll change it and verify. Thank you

Reply:
Seems like it's working :)